modules/402-ingress-nginx/templates/kruise/webhook.yaml

```diff
@@ -72,7 +72,7 @@ webhooks:
     admissionReviewVersions:
       - v1
     sideEffects: None
-    timeoutSeconds: 30
+    timeoutSeconds: 28
```

Comment:
```suggestion
    # set to this value because of DigitalOcean K8s Upgrade requirements (hooks' timeouts should be less than 30 seconds)
    timeoutSeconds: 28
```